app/services/resend_deregistration_email_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ResendDeregistrationEmailService < WasteExemptionsEngine::BaseService
+  attr_reader :registration
+
+  def run(registration:)
+    @registration = registration
+
+    WasteExemptionsEngine::Registration.transaction do
+      registration.update!(deregistration_email_sent_at: nil)
+      SendDeregistrationEmailJob.perform_later(registration.id)
+    end
+
+    true
+  rescue StandardError => e
+    Airbrake.notify e
+    Rails.logger.error "Resend deregistration email for registration #{registration.id}:\n#{e}"
```

Comment:
`registration.reg_identifier` might be more useful for debugging; also send to Airbrake